src/ui/dialogs/scenesettingsdialog.h

```diff
@@ -75,6 +75,7 @@ class UI_EXPORT SceneSettingsDialog : public QDialog
 
 private:
     bool validate();
+    void updateDuration();
```

Comment:
I recommend taking the index offered by `currentIndexChanged`, that way we don't need to ask for it in `updateDuration`.

```
void updateDuration(int index);
```

Reply:
OK